File: Learn the Basic/Pattern_printing/Inverted_Right_Pyramid.cpp
#include<iostream>
using namespace std;
void Inverted_Right_Pyramid(int n){
    for(int i=1;i<=n;i++){
        for(int j=i;j<=n;j++){
            cout<<"*";
        }
        cout<<endl;
    }
}
int main(){
    int n;
    cout<<"Enter the number of N: ";
    cin>> n;
    Inverted_Right_Pyramid(n);
    return 0;
}

File: Learn the Basic/Pattern_printing/Inverted_Right_Pyramid.py
# Inverted_Right_Pyramid
def Inverted_Right_Pyramid(n):
    for i in range(1,n+1):
        for j in range(i,n+1):
            print("*",end="")
        print()
# call the function
n=int(input("Enter the number of n: "))
Inverted_Right_Pyramid(n)